lib/osf-components/addon/components/registries/schema-block-renderer/editable/files/component.ts

```diff
@@ -99,4 +99,17 @@ export default class Files extends Component {
     onAddFile(addedFile: File) {
         this.select(addedFile);
     }
+
+    @action
+    onDeleteFile(deletedFile: File, options: { callback?: () => void }) {
+        if (deletedFile.isFolder) {
```

Comment:
We're already checking if the item is a folder from when we add the callback. Are you sure that we'd never want a callback if the deletedFile is a file? I don't know any cases off the top of my head, but the double-check seems a bit limiting.

Reply:
Good catch. It makes more sense to check against callback.